tests/sqlparser_common.rs

```diff
@@ -4618,3 +4618,20 @@ fn parse_position_negative() {
         res.unwrap_err()
     );
 }
+
+#[test]
+fn parse_is_boolean() {
+    one_statement_parses_to(
```

Comment:
can you also please add a test for the `false` form -- like `SELECT f from foo where field is false`?